### ISSUE TITLE
add pm when user online plugin

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [Abdur-rahmaanJ/honeybot#3]

**Comment by marceloyb** [comment]
how is this issue? anyone did anything? can i take this?

**Comment by AngeloGiacco** [comment]
I believe this is the .joins plugin which has already been developed. Correct @Abdur-rahmaanJ?

**Comment by Abdur-rahmaanJ** [comment]
@AngeloGiacco no. pm means private message. if user offline, next time user comes online, sends him a pm with the message

**Comment by marceloyb** [comment]
i'll take this then.. @Abdur-rahmaanJ the idea is to msg user if the bot keeps online all the time between the user becomes online again?

**Comment by Abdur-rahmaanJ** [comment]
yes
